src/Components/Form_register/Form_Register.js

```diff
@@ -0,0 +1,155 @@
+import React from 'react'
+import './Form_Reg.css'
```

Comment:
Suy nghĩ cách tổ chức code
```suggestion
import './css/form/register.css'
```
Tên file css không phải 1 component do vậy hãy viết chữ in thường